src/Relation/Binary.agda

```diff
@@ -59,6 +59,25 @@ record Preorder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
 
   open IsPreorder isPreorder public
 
+------------------------------------------------------------------------
+-- Partial Setoids
+
+-- Partial equivalence relations are defined in Relation.Binary.Core.
+-- Furthermore, PartialSetoid is parameterized by a type to make
+-- dealing with subsets more feasible.
+
+record PartialSetoid {c} (Carrier : Set c) ℓ  : Set (suc (c ⊔ ℓ)) where
```

Comment:
I'm wondering if this and `IsPartialEquivalence` should go in a new module `Relation.Binary.Partial`? I'm not too keen on having anything with an explicit `Carrier` in alongside the rest of the hierarchy...

Reply:
I like that idea.